packages/engine/Source/Scene/VoxelEllipsoidShape.js

```diff
@@ -166,100 +164,68 @@ VoxelEllipsoidShape.prototype.update = function (
   clipMinBounds,
   clipMaxBounds
 ) {
-  clipMinBounds = defaultValue(
-    clipMinBounds,
-    VoxelEllipsoidShape.DefaultMinBounds
-  );
-  clipMaxBounds = defaultValue(
-    clipMaxBounds,
-    VoxelEllipsoidShape.DefaultMaxBounds
-  );
+  const { DefaultMinBounds, DefaultMaxBounds } = VoxelEllipsoidShape;
+  clipMinBounds = defaultValue(clipMinBounds, DefaultMinBounds);
+  clipMaxBounds = defaultValue(clipMaxBounds, DefaultMaxBounds);
   //>>includeStart('debug', pragmas.debug);
   Check.typeOf.object("modelMatrix", modelMatrix);
   Check.typeOf.object("minBounds", minBounds);
   Check.typeOf.object("maxBounds", maxBounds);
   //>>includeEnd('debug');
 
-  const defaultMinLongitude = VoxelEllipsoidShape.DefaultMinBounds.x;
-  const defaultMaxLongitude = VoxelEllipsoidShape.DefaultMaxBounds.x;
-  const defaultLongitudeRange = defaultMaxLongitude - defaultMinLongitude;
-  const defaultLongitudeRangeHalf = 0.5 * defaultLongitudeRange;
-  const defaultMinLatitude = VoxelEllipsoidShape.DefaultMinBounds.y;
-  const defaultMaxLatitude = VoxelEllipsoidShape.DefaultMaxBounds.y;
-  const defaultLatitudeRange = defaultMaxLatitude - defaultMinLatitude;
-
   const epsilonZeroScale = CesiumMath.EPSILON10;
   const epsilonLongitudeDiscontinuity = CesiumMath.EPSILON3; // 0.001 radians = 0.05729578 degrees
   const epsilonLongitude = CesiumMath.EPSILON10;
   const epsilonLatitude = CesiumMath.EPSILON10;
   const epsilonLatitudeFlat = CesiumMath.EPSILON3; // 0.001 radians = 0.05729578 degrees
 
-  // Clamp the longitude / latitude to the valid range
-  const shapeMinLongitude = CesiumMath.clamp(
-    minBounds.x,
-    defaultMinLongitude,
-    defaultMaxLongitude
-  );
-  const shapeMaxLongitude = CesiumMath.clamp(
-    maxBounds.x,
-    defaultMinLongitude,
-    defaultMaxLongitude
-  );
-  const clipMinLongitude = CesiumMath.clamp(
-    clipMinBounds.x,
-    defaultMinLongitude,
-    defaultMaxLongitude
-  );
-  const clipMaxLongitude = CesiumMath.clamp(
-    clipMaxBounds.x,
-    defaultMinLongitude,
-    defaultMaxLongitude
-  );
-  const renderMinLongitude = Math.max(shapeMinLongitude, clipMinLongitude);
-  const renderMaxLongitude = Math.min(shapeMaxLongitude, clipMaxLongitude);
-
-  const shapeMinLatitude = CesiumMath.clamp(
-    minBounds.y,
-    defaultMinLatitude,
-    defaultMaxLatitude
+  // Don't let the height go below the center of the ellipsoid.
+  const radii = Matrix4.getScale(modelMatrix, scratchScale);
+  const actualMinBounds = Cartesian3.clone(DefaultMinBounds, new Cartesian3());
```

Comment:
Scratch variable for the result?

https://github.com/CesiumGS/cesium/blob/main/Documentation/Contributors/CodingGuide/README.md#result-parameters-and-scratch-variables

This also applies throughout this function.